configure.py

```diff
@@ -939,7 +1249,7 @@ def _run():
         action=argparse.BooleanOptionalAction,
         help="Only download the external dependencies",
     )
-    parser.add_argument("--host", choices=(), default=None, help="Cross compilation host machine")
+    parser.add_argument("--host", choices=("Android",), default=None, help="Cross compilation host machine")
     parser.add_argument(
         "--host-arch",
         choices=("arm", "aarch64", "x86_64"),
```

Comment:
In this commit, with the suggested `_ANDROID_INFO` change mentioned earlier, you can use `_ANDROID_INFO.keys()` for choices